osf/migrations/0100_set_access_request_enabled.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.11 on 2018-04-30 18:34
+from __future__ import unicode_literals
+
+from django.db import migrations, models ,connection
+
+from osf.models import AbstractNode
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0099_merge_20180427_1109'),
+    ]
+
+    def add_default_access_requests_enabled(self, *args, **kwargs):
+        # Get the date the original noderequest migration was applied
+        sql = "SELECT applied from django_migrations WHERE app = 'osf' AND name = '0077_add_noderequest_model';"
+        with connection.cursor() as cursor:
+            cursor.execute(sql)
+            date_noderequest_migration = cursor.fetchall()[0][0]
+
+        # Get all projects created before that
+        AbstractNode.objects.filter(created__lte=date_noderequest_migration).update(access_requests_enabled=True)
```

Comment:
Discussed in person. We can just get all the nodes that have `access_requests_enabled = null`.

We should also exclude registrations.

So the forward migration can be simplified to:

```python
        Node.objects.filter(access_requests_enabled__isnull=True).update(access_requests_enabled=True)
```

 I think the backwards migration can stay as is, though, except changing `AbstractNode` to `Node`.